go/obscuronode/host/l1_client.go

```diff
@@ -1,11 +1,15 @@
-package ethclient
+package host
```

Comment:
No, that wasn't the reason, as the config is itself in its own package.

In a way, this felt to me like it should be part of `host`, because it's how the host connects to an L1 network. But I don't feel strongly, so I'll move it back.

Reply:
Yeah fair, I think the danger would be that host would be a gigantic ball of spaghetti if we put all of its code in the same host package.

If we have part in their own packages eventually then host itself will just be wiring together the `ethclient`, `p2p`, `enclaveclient` etc. and their logic can be self contained. Like by moving that in you added a dependency on geth to host but now it doesn't need to know how we connect to the L1.